packages/extension-bold/test.dom.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Since we rely on `JSOM` to mount an in-memory `DOM` so we can bind the `Editor` to an `HTMLElement` this file is needed as it serves `JSDOM` for the initial `DOM` structure